index.html

```diff
@@ -19,10 +20,188 @@ <h1 class="textdeco1 text1width">José Fernández-Caminero</h1>
+                <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+            </button>
+        </div>
+
+        <div>
+            <img class="fullWidth" src="resources/Rectangle 21(5).png" alt="Another laptop and other objects">
+            <div class="blueBox">
+                <h3 class="projectName">Project name goes here</h3>
+
+                <ul class="greenul">
+                    <li class="greenli">HTML/CSS</li>
+                    <li class="greenli">Ruby on Rails</li>
+                    <li class="greenli">JavaScript</li>
+                </ul>
+
+                <button class="button1 width65 buttonMargin1">See this project
+                    <img src="resources/Union.png" alt="Cool (White) Arrow Pointing Right">
+                    <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+                </button>
+            </div>
+        </div>
+
+        <img class="fullWidth" src="resources/Rectangle 21(6).png" alt="Picture of laptop and other objects">
+        <div class="blueBox">
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="button1 width65 buttonMargin1">See this project
+                <img src="resources/Union.png" alt="Yet An Even Cooler Arrow Pointing Right">
+                <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+            </button>
+        </div>
+    </section>
+
+    <header>
+        <div>
+            <h1 class="textdeco2">About</h1>
+            
+            <p class="paragraph1">
+                I love to code. I think coding is where the arts and the sciences meet. It's the most natural activity for a renaissance person.
+            </p>
+            
+            <button class="button1 width90">
+                <div class="dflex">
+                    <p class="margRight">Get my CV</p>
+                    <img src="resources/ic_download.svg" alt="Downward Arrow">
+                </div>
+            </button>
+        </div>
+
+        <section>
+            <div class="margTop1">
+                <section class="blueRectangle marginleft17 width90">
+                    <section class="poundsign"></section>
+
+                    <section class="rectangleText">
+                        <h2 class="PoppinsSemiBold">Languages</h2>
+                        <ul>
+                            <li class="PoppinsRegular">JavaScript(ES5/ES6)</li>
+                            <li class="PoppinsRegular">Ruby</li>
+                            <li class="PoppinsRegular">HTML</li>
+                            <li class="PoppinsRegular">CSS</li>
+                        </ul>
+                    </section>
```

Comment:
According to the project requirement:
> <img width="994" alt="Screenshot 2022-10-31 at 18 22 54" src="https://user-images.githubusercontent.com/66112760/199070028-62a2f446-d908-4bff-b0ed-6c8bf68508a0.png">

I suggest you use nested `ul` and `li` tags to list out the sections, you can check for an example below: 🙂 

```
<ul>
    <li>
         <ul>
            <li>Languages</li>
            <li>JavaScript (ES5/ES6)</li>
            <li>Ruby</li>
            <li>HTML</li>
            <li>CSS</li>
        </ul>
    </li>
    <li>...</li>
    <li>...</li>
</ul>
```